installations/src/ios/installations_ios.mm

```diff
@@ -40,6 +40,12 @@
   // Destructor is necessary for ARC garbage collection.
 }
 
+void InstallationsInternal::LogHeartbeat(const firebase::App& app) {
+  // Calling the native getter is sufficient to cause a Heartbeat to be logged.
+  FIRApp *platform_app = app.GetPlatformApp();
+  [FIRInstallations installationsWithApp:platform_app];
```

Comment:
Is the iOS implementation ready? From this code piece I don't think it's actually doing anything. Anything i'm missing here?

Reply:
The way to trigger heartbeat logging at iOS/Android level is to call their getter and I think [installationsWithApp](https://firebase.google.com/docs/reference/ios/firebaseinstallations/api/reference/Classes/FIRInstallations#/c:objc(cs)FIRInstallations(cm)installationsWithApp:) is the getter for iOS SDK.

We can pull Nick Cook in for confirmation.

Reply:
My understanding is that the iOS implementation of heartbeat logging is currently in a feature branch, not yet in main. So it's correct that this will initially be effectively no-op. 
Nick gave the OK on a similar pull request for auth: https://github.com/firebase/firebase-cpp-sdk/pull/846